src/components/form-events/index.tsx

```diff
@@ -0,0 +1,120 @@
+'use client';
+import { Button, DatePicker, Form, Input, Radio } from 'antd';
+import { useForm, Controller, FieldErrors } from 'react-hook-form';
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import moment from 'moment';
+import { createValidObject } from './validation';
+import { events } from './mock/events';
+import { useFilterCalendarDate } from './hooks/filterCalendarDate';
+
+export const FormEvents = () => {
+    const fields: string[] = [
+        'UserName',
+        'UserNameTelegram',
+        'CalendarDate',
+        'TimeSlot',
+    ];
+
+    const yupSchema = yup.object().shape(createValidObject(fields));
+
+    const form = useForm({
+        defaultValues: {},
```

Comment:
Create a defaultValues object with all your fields and empty values. And pass here.